package/piwebcam/resolutions.txt

```diff
@@ -0,0 +1,16 @@
+# Default Show-me webcam supported resolutions
+#
+# uncomment resolutions if not wanted
```

Comment:
Shouldn’t this be “comment”?

Reply:
Yes, you are right - commited a fix in 19dc199